.vitepress/theme/custom.css

```diff
@@ -41,4 +41,23 @@ img[center] {
   display: block;
   margin-left: auto;
   margin-right: auto;
+<<<<<<< AP-Workflows-Active
 }
+
+h1 {
+  color:#daa04a
+=======
+>>>>>>> main
+}
```

Comment:
Great PR as always :)

For some reason, the conflict markers are still showing. Just need to remove them and you're good to go.

Conflict markers are:
- <<<<<<< AP-Workflows-Active
- =======
- >>>>>>> main

Reply:
Hmm strange. I remember removing em. Maybe I just didn't finalize it?